src/Pages/sunkyungnoh/Login/components/Appstore.scss

```
@@ -0,0 +1,24 @@
+.appStore {
+  width: 350px;
+  p {
```

```suggestion
  width: 350px;
  
  p {
```
새로운 셀렉터 앞에서 한줄 띄워주세요!

---

네!